### PR TITLE
Parse tileset and draw dogs

### PR DESCRIPTION
**Pull request by Torakko**
Tilesets also moved into content directory.

### CODE REVIEW

File: PixelHunter1995/Content/Content.mgcb [hunk 1]
```diff
@@ -97,3 +97,15 @@
 /processorParam:Quality=Best
 /build:Sounds/92734__tj-mothy__slow-sad-tones.wav;Sounds/slow-music.wav
 
+#begin Tileset/dogs.png
```
Comment (Hattes) [inline PixelHunter1995/Content/Content.mgcb:119]: Is this not also an image and should be put under Images?

File: PixelHunter1995/PixelHunter1995.csproj [hunk 2]
```diff
@@ -58,35 +55,15 @@
   <PropertyGroup>
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```
Comment (feha) [inline PixelHunter1995/PixelHunter1995.csproj:61]: 64-bit platform was removed

File: PixelHunter1995/PixelHunter1995.csproj [hunk 3]
```diff
@@ -114,11 +92,14 @@
...
     <Content Include="Content\Sounds\92734__tj-mothy__slow-sad-tones.wav" />
+    <Content Include="Content\Tileset\dogs.png" />
+    <Content Include="Content\Tileset\dogs.tsx">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
     <Content Include="Icon.ico" />
   </ItemGroup>
   <ItemGroup>
     <MonoGameContentReference Include="Content\Content.mgcb" />
-    <None Include=".editorconfig" />
```
Comment (feha) [inline PixelHunter1995/PixelHunter1995.csproj]: and editorconfig

**Review by feha — CHANGES_REQUESTED** [review]
#54 had its changes to csproj removed.

File: PixelHunter1995/SceneLib/SceneParser.cs [hunk 4]
```diff
@@ -36,24 +34,90 @@ public static Scene ParseSceneXml(string sceneXmlPath)
...
+            string name = "";
+            int tileWidth = 0;
+            int tileHeight = 0;
+            int tileCount = 0;
+            int noOfColumns = 0;
+
+            XmlDocument doc = new XmlDocument();
+            doc.Load(tilesetXmlPath);
+
+            foreach (XmlAttribute attribute in doc.DocumentElement.Attributes)
```
Comment (Hattes) [inline PixelHunter1995/SceneLib/SceneParser.cs]: Why loop over the attributes and not just grab them immediately as above with Dogs and the rest?

File: PixelHunter1995/Content/Content.mgcb (hunk 1 above)
Comment (Torakko) [reply PixelHunter1995/Content/Content.mgcb:119]: Isn't it better to separate tilesets and normal images?

File: PixelHunter1995/PixelHunter1995.csproj (hunk 2 above)
Comment (Torakko) [reply PixelHunter1995/PixelHunter1995.csproj:61]: They will be repaired....

File: PixelHunter1995/PixelHunter1995.csproj (hunk 3 above)
Comment (Torakko) [reply PixelHunter1995/PixelHunter1995.csproj]: And this....

File: PixelHunter1995/SceneLib/SceneParser.cs (hunk 4 above)
Comment (Torakko) [reply PixelHunter1995/SceneLib/SceneParser.cs]: Good question...

**Review by feha — APPROVED** [review]

**Pull request merged**